package.json

```diff
@@ -5,13 +5,17 @@
   "repository": "<replace>",
   "author": "",
   "license": "MIT",
+  "engines": {
+    "node": "16.16"
+  },
   "scripts": {
     "start": "npm run copy-localfiles && npm run parallel",
     "parallel": "./node_modules/.bin/run-p watch offline",
     "presetup": "npm run copy-localfiles",
     "offline": "cd ./build && sls openapi generate -f json && npm run openapireplace && sls offline start --noAuth",
```

Comment:
Will need to look at changing this part of the whole running local piece of work